server/async-worker/index.js

```diff
@@ -0,0 +1,65 @@
+const { Op } = require("sequelize");
+const callScraper = require("../scraper");
+const { Mention, User } = require("../models");
+
+// A rough draft of how new mentions would get pulled into the db
+// without using an explicit /POST route as well as using
+// a given user's credentials.
+module.exports = async function asyncWorker() {
+  const SECONDS = 1;
+  const MILLISECONDS = 1000;
+  // TODO refactor this into smaller functions
+  console.log("[Scraper] Calling async scraper");
+
+  const users = await User.findAll();
+  for (let user of users) {
+    let mentions = await callScraper(user.company);
+    let count = 0;
+
+    for (let m of mentions) {
+      let col, val, timeConvention;
+      // if this is a twitter mention
+      if (m.platform == "Twitter") {
+        col = "content";
+        val = m.content;
```

Comment:
To get rid of this if-else and process mentions from different platform uniformly, we need to discuss how to deal with the problem that tweets don't have a title.